.eslintrc.js

```diff
@@ -1,25 +1,35 @@
 module.exports = {
-  extends: ['rollup', 'plugin:import/typescript'],
+  extends: ['rollup'],
```

Comment:
why is this change necessary? I'm using virtually the same config in several projects without issue.

Reply:
I factored out all TS-related ESLint stuff to one "override" block that only operates on .ts/.tsx. So the "extends" has import/typescript in there instead of the general JS config

Reply:
If you want it in both JS and TS I could set that, but it feels like it belongs in only TS?

Reply:
I'd prefer to revert this change and take a look at the merits of modifying it in an issue, followed by a repo-level commit if we want to move in that direction. The reason being is that this will effect all open PRs and development for all plugins moving forward, and I'm not ready to commit to that big a change.

Reply:
Yeah no worries. It just seemed to make sense at the time but I have no preference. Reverted the change

Reply:
my apologies if it was unclear, but I was speaking to all of the changes in the file. We're totally open to discussing a refactor of the config in another pipeline.

Reply:
Oh - that was not clear yeah.

I brought these changes in because the build fails without them.

From @NotWoods 

> Looks like a linting error is causing the CI failure:
> 
> ```
> > @rollup/plugin-node-resolve@8.1.0 lint:js /home/circleci/project/packages/node-resolve
> > eslint --fix --cache src test types --ext .js,.ts
>  
> /home/circleci/project/packages/node-resolve/test/fixtures/extensions/main.js
>   0:0  error  Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser.
> The file does not match your project config: packages/node-resolve/test/fixtures/extensions/main.js.
> The file must be included in at least one of the projects provided
> 
>  
> ✖ 1 problem (1 error, 0 warnings)
>  
> ```

This is how I limit the TS parser to only TS files (https://github.com/typescript-eslint/typescript-eslint/issues/1928). I'll look into doing it another way tomorrow

Reply:
I see, sorry that I missed that. You can place an overriding `.eslintrc.js` file in that test directory to modify that behavior.  we've done that often in the repo and is perfectly acceptable. If there are too many errors to contend with, you can add that to the `.eslintignore` file. Fixtures always get wiggle room.